client/my-sites/upgrades/index.js

```diff
@@ -14,95 +14,62 @@ const controller = require( 'my-sites/controller' ),
 	paths = require( './paths' ),
 	adTracking = require( 'analytics/ad-tracking' );
 
-module.exports = function() {
-	SiftScience.recordUser();
+function registerMultiPage( { routes, handlers } ) {
+	routes.forEach( path => page( path, ...handlers ) );
```

Comment:
It might be the lovely fewer I'm experiencing right now, but it took me a second to realize that `path` comes from iterating `routes`. I think it'd be better to keep the naming more uniform - iterating `routes` gives you a `route`, not a `path` (or you can go `paths` -> `path`, I'm fine either way).